cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java

```diff
@@ -155,13 +157,21 @@ public void initialize(TwillContext context) {
       CommandLine cmdLine = parseArgs(context.getApplicationArguments());
 
       // Loads configurations
-      hConf = new Configuration();
+      hConf = HBaseConfiguration.create();
+      String numClientRetries = hConf.get(Constants.HBase.CLIENT_RETRIES);
```

Comment:
In fact missing the copying the hConf is a blocker bug as under high load, application will get submitted to the wrong queue.